toolchain/check/testdata/class/partial.carbon

```diff
@@ -82,6 +103,10 @@ library "[[@TEST_NAME]]";
 base class C { }
 
 //@dump-sem-ir-begin
+// CHECK:STDERR: todo_fail_duplicate.carbon:[[@LINE+4]]:9: error: `partial` applied to final type `partial C` [PartialOnFinal]
```

Comment:
Hmm. Is `partial C` always a final type? I suppose it is!

I'd expect you get the same kind of error if you try to derive from `partial C` too?